conbench/api/benchmarks.py

```diff
@@ -6,7 +6,7 @@
 from ..api._endpoint import ApiEndpoint, maybe_login_required
 from ..entities._entity import NotFound
 from ..entities.benchmark_result import (
-    BenchmarkFacadeSchema,
+    BenchmarkResultFacadeSchema,
```

Comment:
Note that personally I am not sure of the information content in `Facade`. I kept it here for now, although maybe it is not useful in the future. Does this try to indicate the JSON surface?